__tests__/fixtures.js

```diff
@@ -81,7 +81,7 @@ const FIXTURE_LOCALSTORAGE_EMPTY = {
   },
   password: '',
   networkNodes: { Ethereum: { selectedNode: 'web3' }, Ropsten: { selectedNode: 'web3' } },
-  mtime: 1591264444786
+  mtime: 1581530607024
```

Comment:
Why? 😆

Reply:
All of our tests use strict equal but the `mtime` fields don't match up between some fixtures which breaks the tests, so i just copied the same time into them all xD